hera_cal/firstcal.py

```diff
@@ -383,7 +387,7 @@ def flatten_reds(reds):
         freds += r
     return freds
 
-def UVData_to_dict(uvdata_list, filetype='miriad'):
+def UVData_to_dict(uvdata_list, filetype='miriad', ex_ants=[]):
```

Comment:
This shouldn't be necessary for firstcal. The ex_ants are already in the info object.